rdrf/rdrf/settings.py

```diff
@@ -748,3 +748,9 @@
 GRAPHENE = {
     "MIDDLEWARE": ["graphene_django.debug.DjangoDebugMiddleware", ]
 }
+
+# XNAT Integration
+# If patient consent is required to lookup xnat results, specify ConsentQuestion item as identified by its code
+XNAT_API_USERNAME = env.get('xnat_api_username', "")
```

Comment:
I think we might need a way to turn xnat off even if the credentials are available. Maybe a registry feature?

Reply:
Added RegistryFeatures.XNAT_INTEGRATION, which is required to enable calls to the xnat api